src/pip/_internal/req/req_install.py

```diff
@@ -693,6 +765,7 @@ def _clean_zip_name(self, name, prefix):  # only used by archive.
 
     # TODO: Investigate if this should be kept in InstallRequirement
     #       Seems to be used only when VCS + downloads
+    # TODO: Consider PEP 517 implications
```

Comment:
There's a new TODO here. Pointing it out in case you missed it.

Reply:
Yeah, I don't *think* there's anything to do but I'll take a look - thanks, I had forgotten this one.

Reply:
Nothing needed here, so I removed the TODO